oozie-to-airflow/converter/primitives.py

```diff
@@ -42,17 +42,25 @@ class Workflow:  # pylint: disable=too-few-public-methods
     nodes: Dict[str, "parsed_node.ParsedNode"]
     dependencies: Set[str]  # TODO: Check is set likely maintain insertion order (Python 3.6 ?)
```

Comment:
Those should be removed as discussed in #180 . But we can do it in a separate PR /refactor most likely.